android/KMEA/app/src/main/java/com/tavultesoft/kmea/BaseActivity.java

```diff
@@ -18,6 +20,20 @@
 import java.util.Locale;
 
 public class BaseActivity extends AppCompatActivity {
+  static ContextWrapper localeUpdatedContext;
+
+  /**
+   * Some classes aren't an AppCompatActivity and need this helper to localize Toast notifications
+   * in the updated locale.
+   * @param defaultContext - the context to fallback if localeUpdatedContext is null
+   * @param resID - resource ID of the string
+   * @param duration - length of the Toast notification (Toast.LENGTH_LONG or Toast.LENGTH_SHORT)
+   */
+  public static void makeToast(Context defaultContext, int resID, int duration) {
+    Context context = (localeUpdatedContext != null) ? localeUpdatedContext : defaultContext;
+    String msg = context.getString(resID);
+    Toast.makeText(context, msg, duration).show();
+  }
```

Comment:
```suggestion
  /**
   * Some classes aren't an AppCompatActivity and need this helper to localize Toast notifications
   * in the updated locale.
   * @param defaultContext - the context to fallback if localeUpdatedContext is null
   * @param resID - resource ID of the string
   * @param duration - length of the Toast notification (Toast.LENGTH_LONG or Toast.LENGTH_SHORT)
   * @param args - format parameters for the string
   */
  public static void makeToast(Context defaultContext, int resID, int duration, Object... args) {
    Context context = (localeUpdatedContext != null) ? localeUpdatedContext : defaultContext;
    String msg = context.getString(resID);
    Toast.makeText(context, String.format(msg, args), duration).show();
  }
```

Reply:
Thanks! That wasn't as painful as I thought.